### ISSUE TITLE
Share Receive address defaults to single app 

### ISSUE THREAD

**Issue by friendsofbitcoin** [Blockstream/green_android#9]
### Description

Share receive address button defaults to single app instead of giving options

### Version

3.0.6

### Steps to reproduce

Click Receive, than click Share Address button

### Expected behaviour

Options to share to different apps like email , whatsapp, text message app and others.

### Actual behaviour

Mine simply defaults to whatsapp which makes for poor UX. Work around is copying to clipboard than opening up email app and pasting address, but this involves more steps. 

2 suggested fixes. Ideal = have a popup appear which indicates "share via" and a list of common apps 
less ideal = in settings allow one to predefine app to share 

### Device or machine

Android 7.1.1

**Comment by domegabri** [comment]
I can see many different options to share using an Android 9. Might be related to Android version  (Green 3.0.6)

**Comment by bitcoinuser** [comment]
I have the same problem.

**Comment by alessandro-saglimbeni** [comment]
@bitcoinuser what's your Android version?

**Comment by RCasatta** [comment]
It's likely you had options to choose whatever apps is available but it has been choosen "use always" and now the app doesn't ask anymore.

To remove default setting go to Settings > App > Green > Advanced > Clear Defaults

**Comment by bitcoinuser** [comment]
> @bitcoinuser what's your Android version?

Android 7.0

**Comment by friendsofbitcoin** [comment]
@RCasatta tried your suggestion under android settings>general>apps> green>open by default

Indicated "no defaults set" thus clear defaults greyed out. 

This is the first time installing green wallet with a new seed setup and not an upgrade, but perhaps the issue is because I'm using testnet.

**Comment by bitcoinuser** [comment]
In my case, when I click share, open this bitcoin lightning wallet https://play.google.com/store/apps/details?id=com.lightning.denryu
I had added this as default to open on sarutobi game.

**Comment by bitcoinuser** [comment]
> In my case, when I click share, open this bitcoin lightning wallet https://play.google.com/store/apps/details?id=com.lightning.denryu
> I had added this as default to open on sarutobi game.

I uninstalled denryu and now Green share is working ok.

**Comment by friendsofbitcoin** [comment]
Ok , I solved the problem by :

Go to Settings.

Go to Apps.

Select the app that is currently the default launcher for a certain file type. (in my case it was whatsapp)

Scroll down to “Launch By Default”.

Tap “Clear Defaults”.

Doesn't exactly look like a problem with green wallet. The only thing is other apps like gallery where I could share a picture did not have this problem before I cleared this default so there is definitely a usability issue where some apps are effected by these setting and others are not in android. So I don't know if its worth fixing or not but this could add more support and usability issues if not addressed.
